timed/reports/views.py

```diff
@@ -56,13 +64,20 @@ class MonthStatisticViewSet(AggregateQuerysetMixin, ReadOnlyModelViewSet):
     ]
 
     def get_queryset(self):
+        """Don't show statistics to customers."""
+        user = self.request.user
+
         queryset = Report.objects.all()
         queryset = queryset.annotate(
             year=ExtractYear("date"), month=ExtractMonth("date")
         )
         queryset = queryset.values("year", "month")
         queryset = queryset.annotate(duration=Sum("duration"))
         queryset = queryset.annotate(pk=F("year") * 100 + F("month"))
+
+        if CustomerAssignee.objects.filter(user=user, is_customer=True).exists():
```

Comment:
Deny access by default, only allow if explicitly needed. This code only forbids to customers, but everyone else sees all the data